vpr/src/route/rr_node.h

```diff
@@ -100,7 +100,7 @@ class t_rr_node {
     short yhigh() const;
     signed short length() const;
 
-    short capacity() const;
+    //short capacity() const; // ESR API This function has been replaced by RRGraphView::node_capacity()
```

Comment:
Since we comment it out does not impact anything, we can safely remove these lines.

Reply:
Sounds good. Done.